invenio_curations/services/service.py

```diff
@@ -24,17 +23,6 @@ def __init__(self, requests_service, **kwargs):
         """Service initialisation as a sub-service of requests."""
         self.requests_service = requests_service
 
-    @property
-    def curation_role(self):
-        """Creates a RDMCuration request and submits it."""
-        role_name = current_app.config.get("CURATIONS_MODERATION_ROLE")
-        return Role.query.filter(Role.name == role_name).one_or_none()
-
-    @property
-    def curation_role_name(self):
-        """Curation role name from config."""
-        return current_app.config.get("CURATIONS_MODERATION_ROLE")
```

Comment:
moved from the service to the extension, where the other properties like these live